### DISCUSSION TITLE
How to revalidate a path if it belongs to a route group?

### DISCUSSION THREAD

**Discussion by xavhan** [vercel/next.js#53250] [App Router]
Let say that i have the following route structure:

```
/app
  - /(pages)
    - /about
      - page.tsx
    - /[slug]
      - page.tsx
```

Documentation says that groups are made only to organizing routes without affecting url structure, 
and revalidation part mentions that it should reflect folder structure and not the url structure.
There is no mention of groups in the revalidation doc.

If I want to revalidate the `[slug]` path will it be `revalidatePath('/[slug]')` or `revalidatePath('/(pages)/[slug]')` or something else?

**Comment by smakman** [comment]
The path will be the actual [slug], for example `contact` for /contact. I can't get the revalidation working for pages with revalidatePath though and have used revalidateTag instead, where I use the slug as tag. 

```js
export default async function Page({ params: { slug }}) {
  const data = await fetch('...', { next: { tags: [slug] } });
  return <h1>{data.title}</h1>
}
```

And then call `revalidateTag('contact')` when you want to revalidate /contact.

**Comment by mikecebul** [comment]
I found "/(page)/[slug]" works when built locally but not when deployed to Vercel. I'm going insane trying to figure this out.
